src/pages/ImportWorkflow/ImportWorkflow.test.ts

```diff
@@ -78,6 +79,15 @@ jest.mock('react-virtualized', () => ({
   AutoSizer: ({ children }) => children({ width: 300 }),
 }));
 
+jest.mock('src/libs/error', () => {
```

Comment:
Adding a type alias for ErrorExports (using typeof import('[path]') pattern) will then let you specify the return type of the arrow function given to jest.mock, giving greatly improved type safety.  The same for the nearby mocks authored here.  See other unit test files for this pattern being used (inconsistenly I know, but it is detailed in the Unit Test TS Guide)